ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -513,10 +513,11 @@ public static void make_ee_relationships()
         //METHOD FILE
         public static string method_toString()
         {
-            return String.Join(System.Environment.NewLine, new string[] {
-                "LoadDeconvolutionResults|deconvolution_file_names\t" + String.Join("; ", Lollipop.identification_files().Select(s => s.filename).ToArray<string>()),
-                "LoadDeconvolutionResults|neucode_labeled\t" + neucode_labeled.ToString(),
-                "CorrectionFactors|correction_file_names\t" + String.Join("; ", Lollipop.calibration_files().Select(s => s.filename).ToArray<string>()),
+            string method = String.Join(System.Environment.NewLine, new string[] {
```

Comment:
I think a better way to do that would be to just have that information in the saved rawexpcom.tsv, and then when you open it the file name is there. We could have a calibrated boolean or something  (right now when you open, it doesn't have the charge states info so you can't see if it was calibrated that way)